sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/AnalysisErrorSuite.scala

```diff
@@ -532,7 +532,7 @@ class AnalysisErrorSuite extends AnalysisTest {
       Seq(a, Alias(InSubquery(Seq(a), ListQuery(LocalRelation(b))), "c")()),
       LocalRelation(a))
     assertAnalysisError(plan, "Predicate sub-queries can only be used" +
-        " in Filter/DeleteFromTable" :: Nil)
+        " in Filter" :: Nil)
```

Comment:
Why there isn't the new `SupportsSubquery`?

Reply:
`SupportsSubquery` is an internal class and end-users don't know what it is. So I just put "a few commands", see https://github.com/apache/spark/pull/26025/files#diff-1d14ac233eac6f233c027dba0bdf871dR608

Reply:
Ok, got it.